packages/catalog-readmodel-writer/src/consumerServiceV1.ts

```diff
@@ -43,21 +43,8 @@ export async function handleMessageV1(
   eservices: EServiceCollection
 ): Promise<void> {
   await match(message)
-    .with({ type: "EServiceAdded" }, async (msg) => {
-      await eservices.updateOne(
-        {
-          "data.id": msg.stream_id,
-        },
-        {
-          $setOnInsert: adaptEserviceToReadModel(
-            msg.version,
-            msg.data.eservice
-          ),
-        },
-        { upsert: true }
-      );
-    })
```

Comment:
It seems this handler is identical to the one of `"ClonedEServiceAdded"`, so they could be kept together.